site/en/docs/privacy-sandbox/attribution-reporting-event-guide/index.md

```diff
@@ -0,0 +1,946 @@
+- [Deduplication](/docs/privacy-sandbox/attribution-reporting-event-guide/#deduplicate-reports) gives you a way to prevent unwanted duplicate reports.
+
+Additionally, a [number of enhancements to the debugging UI in Chrome](https://bugs.chromium.org/p/chromium/issues/list?q=conversion%20component%3AInternals%3EConversionMeasurement%20%22Dev%20Experience%3A%22%20status%3DFixed&can=1) make it easier to understand and debug the API's behavior, in particular when it comes to priorities and report sending.
+
+## Demo!
+
+To generate real event-level reports and see them in your browser, try the [live demo](https://goo.gle/sppi-devrel-eventlevel).
+
+Check out the [source code](https://github.com/GoogleChromeLabs/trust-safety-demo/tree/main/conversion-measurement)
+for this demo. You can
+[fork and customize](https://github.com/GoogleChromeLabs/trust-safety-demo/tree/main/conversion-measurement#fork-and-customize)
+this demo code as needed.
+
+{% Columns %}
+
+{% Column %}
+{% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/O59eUascDKlhrxR7SbJz.png", alt="ALT_TEXT_HERE", width="400", height="300" %}
+{% endColumn %}
+
+{% Column %}
+{% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/u3Gfm4ebwIr9NREfq5eE.png", alt="ALT_TEXT_HERE", width="400", height="300" %}
+{% endColumn %}
+
+{% endColumns %}
+
+## Enable the API in your browser
+
+If you are developing locally or testing out a deployed demo app in your browser, it's enough to enable the API **in your browser**.
+
+{% Details %}
+{% DetailsSummary %}
+
+How to enable the API in your browser
+
+{% endDetailsSummary %}
+
+Enable the API via a _flag_. A flag is a special setting in your browser.
+
+1. Type or paste `chrome://flags/#enable-experimental-web-platform-features` in Chrome's URL bar. This works on mobile, too.
+2. Click **Enabled.**
+3. Restart Chrome.
+4. Ensure that Privacy Sandbox features are turned **on** at `chrome://settings/privacySandbox`. If
+   they're turned off, this would override your flag and disable the Attribution Reporting API.
+
+<figure>
+   {% Img src="image/O2RNUyVSLubjvENAT3e7JSdqSOx1/bKfzMxqL4gpbDDw3NLDc.png", alt="ALT_TEXT_HERE", width="400", height="261" %}
+   <figcaption>Turn on the flag to enable the API in your browser.</figcaption>
+</figure>
+
+{% endDetails %}
+
+## Enable the API for end users (origin trial)
+
+To enable the API for all your site's users⏤except those who have chosen to opt-out of Privacy Sandbox features⏤register your site for an _origin trial_. You can also register for an origin trial if your code is embedded in sites you don't own, as third-party code.
+
+{% Aside 'key-term' %}
+An _origin trial_ (sometimes called an _experiment_) is a way to test a new or experimental web platform feature, and give feedback to the web standards community on the feature.
+Learn more in [Getting started with Chrome's origin trials](/blog/origin-trials/).
+{% endAside %}
+
+### Design your origin trial
```

Comment:
Can you try making the `detailsSummary` an H3 element and see if it's included in the TOC? If yes, you don't need the duplicate subheading

Reply:
It does, but what I'm trying to do is to be able to deep-link to that section (e.g. if someone asks a question and the answer is in this section, I want to be able to send them a link to this section. Or if a reader wants to share a link to that specific section with someone else.)

Right now I made `detailsSummary` an H3 element, but I can't deep-link to it? There's no `#` I can click on its left, like for other titles.

Reply:
I tried both of these, they don't create the `#` for me:

```
{% Details %}
{% DetailsSummary 'h3' %}
How to design your origin trial
{% endDetailsSummary %}
```

```
{% Details %}
{% DetailsSummary 'h3' %}
### How to design your origin trial
{% endDetailsSummary %}
```